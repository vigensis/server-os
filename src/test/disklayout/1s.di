SCSI	disk0	MFG	SSD	100030242816	no	yes
SCSI	disk1	MFG	DISK	600127266816	no	no
USB	key0	USBMFG	USB Key	15737028608	yes	no
