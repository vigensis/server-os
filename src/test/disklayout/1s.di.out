{
	"vdevs": [
		{
			"name": "disk1",
			"vid": "MFG",
			"pid": "DISK",
			"size": "600127266816",
			"solid_state": false
		}
	],
	"capacity": "600127266816",
	"logs": [
		{
			"name": "disk0",
			"vid": "MFG",
			"pid": "SSD",
			"size": "100030242816",
			"solid_state": true
		}
	]
}
